decidim-meetings/app/jobs/decidim/meetings/upcoming_meeting_notification_job.rb

```diff
@@ -7,6 +7,8 @@ class UpcomingMeetingNotificationJob < ApplicationJob
 
       def perform(meeting_id, checksum)
         meeting = Decidim::Meetings::Meeting.find(meeting_id)
+        return if meeting.hidden? || meeting.withdrawn?
```

Comment:
I have a small doubt regarding the way we are performing this... 
We have a guard method, and some other way of guarding that method ... 

maybe it would be better to have something like

```
return if meeting.hidden? || meeting.withdrawn?
return unless verify_checksum(meeting, checksum)

send_notification(meeting)
````

Reply:
I prefer to have all the guards together too if there isn't any good reason for mixing styles. It makes the code a bit cleaner.

Reply:
Good point  :+1: I also support what @alecslupu pointed out above.

Could you take a look @roxanaopr please?